tests/e2e/bin/test-data.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
I'm aware this is a test file but would be nice to document a bit the functions.